test/tools/jsontests/TransactionTests.cpp

```diff
@@ -37,7 +37,6 @@ namespace dev {  namespace test {
 json_spirit::mValue doTransactionTests(json_spirit::mValue const& _input, bool _fillin)
 {
 	json_spirit::mValue v = _input; // TODO: avoid copying and only add valid fields into the new object.
-	TestOutputHelper::initTest(v.get_obj().size());
```

Comment:
I expected a
```
TestOutputHelper outputHelper(v.get_obj().size());
```

Reply:
this line removed because there is a new line in sub functions which call doTransactionTests

Reply:
auto testOutput = dev::test::TestOutputHelper(fileCount);

Reply:
Alright.